go/host/enclave/guardian.go

```diff
@@ -371,12 +371,13 @@ func (g *Guardian) catchupWithL2() error {
 	return nil
 }
 
-func (g *Guardian) submitL1Block(block *common.L1Block, isLatest bool) error {
+func (g *Guardian) submitL1Block(block *common.L1Block, isLatest bool) (bool, error) {
```

Comment:
What's the new return value for? Doesn't seem to be used and seems to be equal to `err == nil`